gapic/src/main/com/google/gapid/Main.java

```diff
@@ -165,24 +168,44 @@ private void uiStartup(Shell shell) {
         shell.getDisplay().asyncExec(() -> showFirstTimeDialog(shell, models, widgets, onStart));
       }
 
+      // TODO: Only for easy gapis crash testing, will delete before check in.
+      window.statusBar.addListener(SWT.MouseDown, $ -> server.disconnect());
+
```

Comment:
For easy testing, a click on the status message composite can trigger a quick gapis losing failure :) This part will be deleted before check in.